phpcs-ruleset.xml.dist

```diff
@@ -25,7 +25,6 @@
         <!-- https://github.com/Gizra/og/issues/545 -->
         <exclude name="DrupalPractice.CodeAnalysis.VariableAnalysis.UnusedVariable" />
         <!-- https://github.com/Gizra/og/issues/548 -->
-        <exclude name="DrupalPractice.Commenting.ExpectedException.TagFound" />
```

Comment:
Let's also remove the comment line for this, it is no longer needed after this is accepted.